### PR TITLE
Updating the readme with license and telegram channel

### PR DESCRIPTION
**Pull request by djworth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
-# cardano-wallet-client
```
Comment (sultan05500) [inline README.md:1]: addr1q8n4vqvfr9gjddrqur25lfe8an67z5f9fn0357twfaan3pnqqk3x7sdd803sukvcyzap6m4sdrvpsrg6x5h39sxgu4sqsjd5e5

**Review by Sultan05500 — COMMENTED** [review]
addr1q8n4vqvfr9gjddrqur25lfe8an67z5f9fn0357twfaan3pnqqk3x7sdd803sukvcyzap6m4sdrvpsrg6x5h39sxgu4sqsjd5e5

File: README.md (hunk 1 above)
Comment (Sultan05500) [inline README.md:1]: addr1q8n4vqvfr9gjddrqur25lfe8an67z5f9fn0357twfaan3pnqqk3x7sdd803sukvcyzap6m4sdrvpsrg6x5h39sxgu4sqsjd5e5

**Review by Sultan05500 — COMMENTED** [review]
Wallet update

**Pull request merged**